src/status_im/contexts/wallet/add_account/confirm_address_to_save/view.cljs

```diff
@@ -0,0 +1,116 @@
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def ^:const sheet-closing-delay 750)
+
+(defn- on-success-confirm-address-to-save
+  [theme]
+  (rf/dispatch [:navigate-back])
+  (debounce/debounce-and-dispatch
+   [:navigate-back]
+   sheet-closing-delay)
+  (debounce/debounce-and-dispatch
+   [:toasts/upsert
+    {:type  :positive
+     :theme theme
+     :text  (i18n/label
+             :t/address-saved)}]
+   sheet-closing-delay))
+
+(defn- on-press-confirm-address-to-save
+  [{:keys [account-name account-color address ens? theme]}]
+  (rf/dispatch
+   [:wallet/save-address
+    {:address             address
+     :name                account-name
+     :customization-color account-color
+     :ens                 (when ens? address)
+     :on-success          #(on-success-confirm-address-to-save theme)}]))
+
+(defn view
+  []
+  (let [theme                                          (quo.theme/use-theme)
+        {:keys                  [adding-address-purpose
+                                 ens? address]
+         {:keys [placeholder
+                 address-type
+                 button-label]} :confirm-screen-props} (rf/sub [:wallet/currently-added-address])
+        [account-name on-change-name]                  (rn/use-state "")
+        [account-color on-change-color]                (rn/use-state (rand-nth colors/account-colors))
+        [selected-networks set-selected-networks]      (rn/use-state (set
+                                                                      constants/default-network-names))]
+    [:<>
+     [rn/view {:style style/save-address-drawer-bar-container}
+      [quo/drawer-bar]]
+     [rn/view {:style style/container}
+      [create-or-edit-account/view
+       {:placeholder         (i18n/label placeholder)
+        :account-name        account-name
+        :account-color       account-color
+        :on-change-name      on-change-name
+        :on-change-color     on-change-color
+        :watch-only?         true
+        :top-left-icon       :i/arrow-left
+        :bottom-action-label button-label
+        :bottom-action-props {:customization-color account-color
+                              :disabled?           (string/blank? account-name)
+                              :accessibility-label :confirm-button-label
+                              :on-press            #(on-press-confirm-address-to-save
+                                                     {:ens?                   ens?
+                                                      :adding-address-purpose adding-address-purpose
+                                                      :account-name           account-name
+                                                      :account-color          account-color
+                                                      :address                address
+                                                      :theme                  theme})}}
```

Comment:
This could also be a `use-callback` if want to avoid extra re-renders for the bottom-action button.